kivy/core/window/__init__.py

```diff
@@ -211,6 +211,10 @@ class WindowBase(EventDispatcher):
             Width of the window.
         `height`: int
             Height of the window.
+        `minimum_width`: int
+            Minimum width of the window
+        `minimum_height`: int
+            Minimum height of the window
```

Comment:
No problem, will add it when I get some spare time over the next couple of
days.

On Fri, Jun 12, 2015 at 2:18 AM, Mathieu Virbel <notifications@github.com>
wrote:

> In kivy/core/window/__init__.py
> <https://github.com/kivy/kivy/pull/3414#discussion_r32284330>:
>
> > @@ -211,6 +211,10 @@ class WindowBase(EventDispatcher):
> >              Width of the window.
> >          `height`: int
> >              Height of the window.
> > +        `minimum_width`: int
> > +            Minimum width of the window
> > +        `minimum_height`: int
> > +            Minimum height of the window
>
> It need a .. versionadded:: 1.9.1 tag, check in the docstring, there
> should be one for other added token as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kivy/kivy/pull/3414/files#r32284330>.
>